src/sensors/odometer/interrupt/DirectionlessOdometer.hpp

```diff
@@ -67,11 +67,10 @@ class DirectionlessOdometer : public Odometer
     virtual void update();
 
 protected:
-    const float
-        mPulsesPerMeterRatio; // NOLINT: Refactor this so protected variables are not necessary
-    volatile unsigned long mPulsesCounter{ 0 }; // NOLINT
-    volatile unsigned long mPreviousPulse{ 0 }; // NOLINT
-    volatile unsigned long mDt{ 0 };            // NOLINT
+    const float mPulsesPerMeterRatio;
```

Comment:
:warning: **cppcoreguidelines-non-private-member-variables-in-classes** :warning:
member variable 'mPulsesPerMeterRatio' has protected visibility